code/iaas/api-logic/src/main/java/io/cattle/platform/iaas/api/lb/LoadBalancerConfigSetListenersActionHandler.java

```diff
@@ -46,7 +54,7 @@ public HandlerResult handle(ProcessState state, ProcessInstance process) {
             // create a new set
             createNewListenerMaps(config, newListenerIds);
         }
-        return null;
+        return objectManager.reload(config);
```

Comment:
Right, it doesn't really seem to need a reload. None of the state on the base resource is changing as a result of these actions.